### PR TITLE
Finish Exercise 6

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

File: customview/Exercise_6/Exercise_6/HomeViewController.swift [hunk 1]
```diff
@@ -2,34 +2,50 @@
...
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        view.endEditing(true)
```
Comment (at-hieutran) [inline customview/Exercise_6/Exercise_6/HomeViewController.swift]: Super ?

File: customview/Exercise_6/Exercise_6/CustomSliderView.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        customSlider()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        customSlider()
+    }
+
+    var circle = UIView()
+    var rectangletop = UIView(frame: CGRect(x: 100, y: 100, width: 20, height: 100))
```
Comment (at-truongnguyen) [inline customview/Exercise_6/Exercise_6/CustomSliderView.swift]: variable name follow `camel rule` please

File: customview/Exercise_6/Exercise_6/HomeViewController.swift [hunk 3]
```diff
@@ -2,34 +2,51 @@
...
+//  Created by Son Vu L.L on 6/16/17.
+//  Copyright © 2017 Son Vu L.L. All rights reserved.
 //
 
 import UIKit
 
 class HomeViewController: UIViewController {
-
+    let customSlider = Bundle.main.loadNibNamed("CustomSliderView", owner: self, options: nil)?.first as! CustomSliderView
+    var textFieldPercent = UITextField()
```
Comment (at-truongnguyen) [inline customview/Exercise_6/Exercise_6/HomeViewController.swift]: percentageTextField please.  :)

File: customview/Exercise_6/Exercise_6/CustomSliderView.swift [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+//  CustomSliderView.swift
+//  Exercise_6
+//
+//  Created by Son Vu L.L on 6/16/17.
+//  Copyright © 2017 Son Vu L.L. All rights reserved.
+//
+
+import UIKit
+protocol CustomSliderViewDelegate {
+    func getCurrentPoint(value: CGFloat)
```
Comment (at-truongnguyen) [inline customview/Exercise_6/Exercise_6/CustomSliderView.swift]: hehe, I think your func name in delegate is not correct 👯

File: customview/Exercise_6/Exercise_6/CustomSliderView.swift [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+//
+
+import UIKit
+protocol CustomSliderViewDelegate {
+    func getCurrentPercentage(value: CGFloat)
+}
+
+class CustomSliderView: UIView {
+
+    var delegate: CustomSliderViewDelegate!
```
Comment (at-truongnguyen) [inline customview/Exercise_6/Exercise_6/CustomSliderView.swift]: `delegate ` can be nil and `weak`

File: customview/Exercise_6/Exercise_6/CustomSliderView.swift [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+//  CustomSliderView.swift
+//  Exercise_6
+//
+//  Created by Son Vu L.L on 6/16/17.
+//  Copyright © 2017 Son Vu L.L. All rights reserved.
+//
+
+import UIKit
+protocol CustomSliderViewDelegate {
+    func getCurrentPercentage(value: CGFloat)
```
Comment (at-truongnguyen) [inline customview/Exercise_6/Exercise_6/CustomSliderView.swift]: Should change it
ex `customSliderView(view: CustomSliderView valueChanged percent: CGFloat)`

File: customview/Exercise_6/Exercise_6/CustomSliderView.swift [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+        circle.layer.cornerRadius = 25
+        rectangleTop.addSubview(circle)
+        rectangleTop.backgroundColor = UIColor.black
+        percentageLabel.center = CGPoint(x: circle.bounds.width / 2, y: circle.bounds.height / 2)
+        circle.addSubview(percentageLabel)
+        percentageLabel.text = "50"
+        percentageLabel.textAlignment = .center
+        percentageLabel.textColor = .white
+        self.addSubview(rectangleDown)
+        self.addSubview(rectangleTop)
```
Comment (at-truongnguyen) [inline customview/Exercise_6/Exercise_6/CustomSliderView.swift]: remove self please

**Pull request merged**